clients/fides-js/src/components/fides.css

```diff
@@ -53,18 +53,20 @@
 
   /* Everything else */
   --fides-overlay-font-family: Inter, sans-serif;
-  --fides-overlay-font-size-body-small: 0.875em;
-  --fides-overlay-font-size-body: 0.95em;
-  --fides-overlay-font-size-title: 1em;
-  --fides-overlay-font-size-buttons: 1.05em;
-  --fides-overlay-padding: 1.5em;
+  --12px: 0.75rem;
+  --14px: 0.875rem;
+  --15px: 0.9375rem;
+  --16px: 1rem;
```

Comment:
this strategy is from https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#leveraging-css-variables-15